docs/integrations/sources/snapchat-marketing.md

```diff
@@ -17,6 +17,10 @@ This page guides you through the process of setting up the Snapchat Marketing so
 * client_secret
 * refresh_token
 * start_date
+* end_date
+* action_report_time (Optional)
```

Comment:
Please add more details for what those param responsible for and what is default value for them, even it's empty.

Reply:
Done